src/Root/Finish.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { db } from '../firebase-config';
+import ResumeOrder from '../components/resumeOrder/ResumeOrder'
+import './finish.css'
+
+function Finish (){
```

Comment:
si este componente finaliza la orden tiene que tener bien definido el nombre
```suggestion
function FinishOrder (){
```